test/local/password_routes.js

```diff
@@ -84,6 +86,52 @@ test(
 )
 
 test(
+  '/password/forgot/verify_code',
+  function (t) {
+    var mockCustoms = {
+      check: sinon.spy(function () {
+        return P.resolve()
+      })
+    }
+    var uid = uuid.v4('binary')
+    var mockDB = mocks.mockDB({
+      email: TEST_EMAIL,
+      passCode: 'abcdef',
+      passwordForgotTokenId: crypto.randomBytes(16),
+      uid: uid
+    })
+    var passwordRoutes = makeRoutes({
+      customs: mockCustoms,
+      db: mockDB
+    })
+
+    var mockRequest = mocks.mockRequest({
+      credentials: {
+        email: TEST_EMAIL,
+        passCode: Buffer('abcdef', 'hex'),
+        ttl: function () { return 17 },
+        uid: uid
+      },
+      payload: {
+        code: 'abcdef'
+      },
+      query: {}
+    })
+    return new P(function(resolve) {
+      getRoute(passwordRoutes, '/password/forgot/verify_code')
+        .handler(mockRequest, resolve)
+    })
+    .then(function(response) {
```

Comment:
Testing locally, response returned is an error `mailer.sendPasswordResetNotification is not a function`. I think using the mock mailer in here will resolve these. Also, check that response has a valid `accountResetToken`.